### ISSUE TITLE
Add PayPal to the payments

### ISSUE THREAD

**Issue by Fetati** [Goodler/code#436]
Add PayPal to the payments.
People ask me. 

"I created my account, but not so comfortable to enter my CC info within the website. 
Do you have plan to integrate with Paypal sometime soon?"

**Comment by DeveloperRegistry** [comment]
Will do. Please initiate creation of a merchant account for Gooddler with PayPal.

**Comment by galinafed** [comment]
We have now Gooddler account on paypal

**Comment by DmitrysIF** [comment]
Принято.

**Comment by DmitrysIF** [comment]
Есть прототип, проверяется работа в составе проекта. После испытаний будет полностью переделываться логика платёжной системы GOODdler, изменение её архитектуры в сторону упрощения сопровождения и понимания, а так же улучшения качества кода.

**Comment by DmitrysIF** [comment]
Прототип paypal перенесён на новую архитектуру, тестируется. После проверки передам Антону на ревью и по итогам будет решаться вопрос Релиза платежей с помощью paypal.

**Comment by DmitrysIF** [comment]
@Fetati , @galinafed Нам необходим шаблон письма для платежей, которые были осуществлены с помощью PayPal. Тот, который есть при оплате карточкой - не подходит по данным.

**Comment by Fetati** [comment]
Что ты имеешь ввиду, @DmitrysIF . Это письмо после того, как люди оплатили через PayPal? Thank you letter?

**Comment by DmitrysIF** [comment]
Да, как здесь: https://github.com/Goodler/code/issues/461, только адаптированную под paypal 
@Fetati

**Comment by DmitrysIF** [comment]
http://0a9e2d8be7274a70ab78a987e20d16b2.cloudapp.net/ гуддлер с пейпалом.
Подключен мой тестовый аккаунт, все "платежи" с помощью него на данный момент будут проходить в "песочнице" пейпала. Для того, чтобы сделать тестовый платёж, воспользуйтесь следующими данными:
Логин пейпал - testemailpersonal@mail.ru
Пароль пейпал - 12345678

**Comment by Fetati** [comment]
Я не могу войти по линку. Мой пароль-имейл не подходят

**Comment by DmitrysIF** [comment]
Это тестовая база данных. Аккаунты с релизной базы данных работать не будут. Нужно или создать новый тестовый аккаунт, или найти данные от старого тестового аккаунта.

**Comment by DmitrysIF** [comment]
@Fetati , @galinafed Вы уже ознакомились с тем, как сейчас выглядит процесс платежа с пейпал?

**Comment by Fetati** [comment]
Я не могу войти. Мой имейл, пишет, что уже существует. С моим существующим паролем очень долго пытается войти, а потом ругается, и выкидывает.
Вот такое состояние может быть минут 15 и больше:
![screen shot 2015-07-03 at 12 15 49 am](https://cloud.githubusercontent.com/assets/4335901/8494327/cadc8c42-2118-11e5-977b-6a8a99d23be6.png)

**Comment by Fetati** [comment]
Все мои существующие имейлы уже зарегистрированы, я не помню паролей. А тот, который помню, не открывается. Есть другие пути войти в тестовый эккаунт? Может почистить мои имейлы, чтобы я заново зарегистрировалась?

**Comment by DmitrysIF** [comment]
Можно зарегистрировать несуществующий емейл, как мы обычно делаем в тестовых целях.
Могу дать ещё свой тестовый аккаунт.
login: dmitrys@ifface123.com
password: test_User

Ещё я могу попробовать скопировать аккаунт из релизной базы данных в тестовую, и тогда будет один аккаунт будет работать везде с одним и тем же паролем (укажите емейл нужного аккаунта).

**Comment by DmitrysIF** [comment]
Попробуй сейчас войти по своему емейлу fedorova.tanya@gmail.com

**Comment by Fetati** [comment]
Дима, можешь, плиз, вот так вот разместить кнопку paypal. И под кнопкой вставить OR

![paypal_sample](https://cloud.githubusercontent.com/assets/4335901/8538646/9cfda0a4-2424-11e5-954d-0b767cb59057.jpg)

**Comment by DmitrysIF** [comment]
Хорошо, будет сделано!

**Comment by DmitrysIF** [comment]
@Fetati , @galinafed , @DeveloperRegistry  Прошу сообщать о всех проблемах с paypal здесь, а то у нас скоро может возникнуть путаница :)

**Comment by galinafed** [comment]
May i please get notifications about purchases on galina@gooddler.com

**Comment by DmitrysIF** [comment]
Сейчас проверю базы на предмет наличия Админских прав данного емейла.

**Comment by DmitrysIF** [comment]
@galinafed Такой пользователь не зарегистрирован в гуддлере. А емейлы о платежах рассылаются только админам гуддлера. Зарегистрируйся с этим аккаунтом и сообщи здесь - дам права Админа гуддлера.

**Comment by galinafed** [comment]
i think you can use info@gooddler.com I get those emails

**Comment by DmitrysIF** [comment]
Так, прощу прощения, возможно я ошибся и у нас сейчас нет возможности произвольно задать емейлы, по которым рассылаются уведомления о том, что был сделан платёж. Сейчас уведомления получают емейлы, на которые были зарегистрированы платёжные аккаунты гуддлера. Вопрос требует углубленных консультаций.

**Comment by galinafed** [comment]
no problem, will ask Tatiana to keep forwarding those to me))

**Comment by galinafed** [comment]
or, am i on paypal as galinafed@gmail.com? This email will also work

**Comment by DmitrysIF** [comment]
Возможно, сейчас единственный выход, который я вижу - включить перенаправление емейлов на твой адрес средствами почтового ящика, на который зарегистрирован аккаунт гуддлер пейпала.

**Comment by DmitrysIF** [comment]
У нас, на данный момент, нет нотификаций с гуддлера о совершённых платежах админам. Только клиентам, которые сделали покупку. Нужно обсудить актуальность и важность такой возможности для админов. Нужно подождать Антона.

**Comment by DmitrysIF** [comment]
@galinafed Я поговорил с Антоном и он просит написать требования к системе оповещения админов о совершённых платежах пользователями гуддлера и предоставить шаблон письма для этого сообщения. Лучше всего создать новый Issues с этой задачей.
